test/Altinn.Profile.Tests/IntegrationTests/UserProfileTests.cs

```diff
@@ -144,6 +144,139 @@ public async Task GetUsersById_SblBridgeFindsProfile_ResponseOk_ReturnsUserProfi
+
+            string responseContent = await response.Content.ReadAsStringAsync();
+
+            UserProfile actualUser = JsonSerializer.Deserialize<UserProfile>(
+                responseContent, serializerOptionsCamelCase);
+
+            // These asserts check that deserializing with camel casing was successful.
+            Assert.Equal(userId, actualUser.UserId);
+            Assert.Equal(userUuid, actualUser.UserUuid);
+            Assert.Equal("LEO WILHELMSEN", actualUser.Party.Name);
+            Assert.Equal(userUuid, actualUser.Party.PartyUuid);
+            Assert.Equal("LEO", actualUser.Party.Person.FirstName);
+            Assert.Equal("nb", actualUser.ProfileSettingPreference.Language);
+        }
+
+        [Fact]
+        public async Task GetUsersByUuid_SblBridgeFindsProfile_NotAuthorized_ReturnsForbidden()
+        {
+            // Arrange
+            const int userId = 20000009;
+            Guid userUuid = new("cc86d2c7-1695-44b0-8e82-e633243fdf31");
+
+            HttpRequestMessage httpRequestMessage = CreateGetRequest(userId, $"/profile/api/v1/users/{userUuid}");
+
+            HttpClient client = _webApplicationFactorySetup.GetTestServerClient();
+
+            // Act
+            HttpResponseMessage response = await client.SendAsync(httpRequestMessage);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.Forbidden, response.StatusCode);
+        }
+
+        [Fact]
+        public async Task GetUsersByUuid_SblBridgeReturnsNotFound_ResponseNotFound()
+        {
+            // Arrange
+            const int userId = 20000009;
+            Guid userUuid = new("cc86d2c7-1695-44b0-8e82-e633243fdf31");
+
+            HttpRequestMessage sblRequest = null;
+            DelegatingHandlerStub messageHandler = new(async (request, token) =>
+            {
+                sblRequest = request;
+
+                return await Task.FromResult(new HttpResponseMessage() { StatusCode = HttpStatusCode.NotFound });
+            });
+            _webApplicationFactorySetup.SblBridgeHttpMessageHandler = messageHandler;
+
+            HttpRequestMessage httpRequestMessage = CreateGetRequest(userId, $"/profile/api/v1/users/{userUuid}");
+
+            httpRequestMessage.Headers.Add("PlatformAccessToken", PrincipalUtil.GetAccessToken("ttd", "unittest"));
+
+            HttpClient client = _webApplicationFactorySetup.GetTestServerClient();
+
+            // Act
+            HttpResponseMessage response = await client.SendAsync(httpRequestMessage);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
+
+            Assert.NotNull(sblRequest);
+            Assert.Equal(HttpMethod.Get, sblRequest.Method);
+            Assert.EndsWith($"users?useruuid={userUuid}", sblRequest.RequestUri.ToString());
+        }
+
+        [Fact]
+        public async Task GetUsersByUuid_SblBridgeReturnsForbidden_NotAuthorized()
```

Comment:
Same mistake as in the test above. There are no calls to SblBridge. 

Suggested change. Add a valid AccessToken to the request. That way you will trigger the Bearer token validation. Which is missing. That should give NotAuthorized and not Forbidden. I hope, because then I've learned something today.

Reply:
Nver mind, that would make the test a duplicate of the test below.

Reply:
They do not test SblBridge they test Authorization so no SblBridge is ever reached when the code is working as expected